### DISCUSSION TITLE
Add REGEX to filters

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17018] [Feature improvement]
Been asked by small AI related company to help deploying system for the purpose of data annotation.
Noticed that if you guys add REGEX conditions to filters and workflows it will extend flexibility for miles and miles.
How about that?

Garry

**Comment by mariuszkrzaczkowski** [comment]
give an example of how it would work?

**Reply by garryatbrighton** [reply]
hide all characters excluding digits and dots.
500k records such:
prefixA1887.838.011137
prefixB 5672.093-22.66
show only those records what contains ONLY numbers (whatever numbers) but non of other ASCII chars

**Reply by garryatbrighton** [reply]
basic would be hiding all records what contains any of alphabet letter. Impossible to do it with standard conditions

**Comment by s2b-git** [comment]
I was thinking of it too (or similar) , mostly considering mass edit as to replace part of text string in column rather than whole field.
In the past I had amazing extension in vtiger which does it. Such functionality is based or regex. Screenshot below
![image](https://user-images.githubusercontent.com/84110093/233813888-bd36f2da-b0c5-4813-9886-5d1c24f6a9f7.png)

**Reply by mariuszkrzaczkowski** [reply]
interesting idea
